### ISSUE TITLE
[OUDS] [TRACKING] Backlog to be functionally equivalent to Boosted

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/Orange-Boosted-Bootstrap#2589]
### Description

This tracking issue can be seen as a backlog of things to develop so that OUDS Web is functionally equivalent to Boosted in terms of features, documentation, packaging, and many other things.
Most of these elements have been removed temporarily when creating the branch for OUDS Web; they existed before in Boosted or Bootstrap, and will probably need to be reintegrated at some point. At least, if they don't, we must decide explicitly that they shouldn't be integrated.

### Ready to dev

- [ ] Make Algolia and search work #3299 
  - Might be an issue caused by `site/assets/js/search.js` with the `liveUrl` targeting `web.unified-design-system.orange.com`? 
- [ ] Determine what we do with Meteor define in `package.js`. Do we keep it? Do we need this `package.js` file?
- [ ] Take into account this comment: https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2647#pullrequestreview-2196537994

### Storybook

> [!WARNING]  
> Blocked by web.unified-design-system.orange.com not being properly set-up

- [ ] Error while rendering https://orange-opensource.github.io/Orange-Boosted-Bootstrap/ouds-web/storybook. Due to `.storybook/preview-head.html` configured to use `https://web.unified-design-system.orange.com/docs/0.0/assets/css/docs.css`. Should be fixed automatically.
- [x] Reintegrate `ods.web` parameter in `hugo.yml` with the new value

### Documentation

> [!NOTE]  
> As soon as `boosted-sprite.svg` must be reintegrated, mention it in the `NOTICE.txt`

- [x] Redirection of pages to double-check. (:warning: Blocked by web.unified-design-system.orange.com not being properly set-up)
- [ ] Add a page that explains how we are compatible with Bootstrap (maybe the README too). What version is compatible with what Bootstrap version. That the framework is compatible with Bootstrap. etc. 
- [ ] **Homepage**
  - [x] **(On hold: Question asked to the project, waiting for an answer)** Introduce a new OUDS logo replacing Boosted homemade logo? https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/980869e8a3338afa5ed8986c40b1b7dfdb916f4d/site/assets/scss/_boosted.scss#L20-L42
  - [ ] Jumbotron (compare Boosted / Bootstrap)
  - [ ] Get started any way you want (compare Boosted / Bootstrap)
  - [ ] Read our getting started guides (compare Boosted / Bootstrap)
  - [ ] Customize everything with Sass (compare Boosted / Bootstrap)
  - [ ] Build and extend in real-time with CSS variables (compare Boosted / Bootstrap)
  - [ ] Components, meet the Utility API (compare Boosted / Bootstrap)
  - [ ] Powerful JavaScript plugins without jQuery (compare Boosted / Bootstrap)
  - [ ] **Orange Design System for web** section
    - [ ] Reintegrate `site/layouts/partials/icons/download.svg` and `site/static/docs/**/assets/img/boosted-guideline*` images and mention them in the `NOTICE.txt` file.
- [ ] **Solaris icons link** in the header
- [ ] Accessibility statement (see https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2665)
- [x] How to render "Bootstrap compatibility" areas? See https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2687/files#r1760593662

### About

- [ ] **About > Brand** ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/about/brand.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/about/brand.md))
  - [ ] Do we reintegrate the OBS logos? If so, reintegrate `site/static/docs/**/assets/brand/OBS-*` and mention them in the `NOTICE.txt` file. We probably will need to provide logos.
  - [ ] `site/static/docs/**/assets/brand/orange-logo-formatted.svg` must be reintegrated and mentioned in the `NOTICE.txt` file
- [ ] **About > Overview**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/about/overview.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/about/overview.md))

### Content

- [ ] **Content > Figures**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/content/figures.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/content/figures.md))
- [ ] **Content > Images**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/content/images.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/content/images.md))
- [x] **Content > Reboot**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/content/reboot.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/content/reboot.md))
  - [x] Replace  `opacity-*` by semantic name in horizontal rules
  - [x] Handle comment whenever lists are developed
  - [x] Handle comment whenever links are developed
  - [x] Handle comment whenever dividers are developed
- [ ] **Content > Tables**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/content/tables.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/content/tables.md))
- [ ] **Content > Typography**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/content/typography.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/content/typography.md))
  - [x] Handle comment whenever lists are developed
  - [ ] Handle comment whenever links are developed

### Customize

- [ ] **Customize > Color modes**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/customize/color-modes.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/customize/color-modes.md))
  - [ ] Handle comment whenever `/components/orange-navbar` is added
  - [ ] Handle comment whenever `/components/footer` is added
  - [ ] Handle comment whenever `/customize/color-theme` is added
- [ ] **Customize > Color palette**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/customize/color-palette.md))
  - [ ] Handle comments whenever `/customize/color-theme` is added
  - [x] Check at some point if we need to change the URL to the design system (see https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2802#discussion_r1879604583)
- [ ] **Customize > Color theme**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/customize/color-theme.md))
- [ ] **Customize > Color** ???: ([Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/customize/color.md))
- [ ] **Customize > Components**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/customize/components.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/customize/components.md)) #3300 
- [ ] **Customize > CSS variables**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/customize/css-variables.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/customize/css-variables.md)) #3301
- [ ] **Customize > Optimize**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/customize/optimize.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/customize/optimize.md))
- [ ] **Customize > Options**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/customize/options.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/customize/options.md))
- [ ] **Customize > Overview**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/customize/overview.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/customize/overview.md))
- [ ] **Customize > Sass**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/customize/sass.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/customize/sass.md))

### Dark mode

- [x] **Dark mode**:  ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/dark-mode.md))

### Extend

- [ ] **Extend > Approach**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/extend/approach.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/extend/approach.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2650
  - [x] Handle comment whenever `.d-none` and `.d.md-none` classes, and responsive concepts are there
  - [ ] Handle comment whenever buttons are developed
  - [ ] Handle comment whenever z-index is developed
  - [ ] Handle comment whenever /getting-started/javascript#data-attributes is there
  - [x] Handle comment whenever utilities such as `.d-block` are there
- [ ] **Extend > Icons**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/extend/icons.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/extend/icons.md))
  - If `site/static/docs/**/assets/img/boosted-warning.svg` needs to be reintegrated, mention it in the `NOTICE.txt` file
  - `icons*` properties from `hugo.yml` were used in this page. If not anymore, drop them from `hugo.yml`. 

### Getting Started

- [ ] **Getting Started > Accessibility**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/getting-started/accessibility.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/getting-started/accessibility.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2675 
  - [ ] Uncomment whenever modal, dialog, dropdown, tooltips are available in "Interactive components"
  - [x] Uncomment "Color contrast" section whenever colors are defined and implemented
  - [x] Uncomment code in "Visually hidden content" whenever `.text-*` are implemented
  - [ ] Uncomment in "Reduced motion" when modal, carousels, and spinners are implemented
  - [ ] Uncomment in "Focus visibility" when carousel is implemented
  - [ ] Uncomment "Under a fixed header" when sticky or fixed headers are implemented
  - [ ] Uncomment in "Maximum line length" when `/utilities/text` is implemented (See #2851)
- [x] **Getting Started > Best practices**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/getting-started/best-practices.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/getting-started/best-practices.md)).
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2657  
- [ ] **Getting Started > Browsers devices**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/getting-started/browsers-devices.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/getting-started/browsers-devices.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2684 
  - [ ] Uncomment elements whenever modals are developed
  - [ ] Uncomment elements whenever dropdowns are developed
- [ ] **Getting Started > Contents**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/getting-started/contents.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/getting-started/contents.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2685
  - [x] Uncomment elements whenever layout grid page is developed (`/layout/grid`)
  - [ ] Uncomment elements whenever content reboot is developed (`/content/reboot`) (See #2851)
- [x] **Getting Started > Contribute**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/getting-started/contribute.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/getting-started/contribute.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2646
- [ ] **Getting Started > Download**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/getting-started/download.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/getting-started/download.md))
- [ ] **Getting Started > Introduction**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/getting-started/introduction.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/getting-started/introduction.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2658 
  - [ ] Uncomment and update the "About Orange Brand" section
  - [x] Uncomment when `/getting-started/contents` is available
  - [ ] Uncomment when the first components are available
  - [ ] Uncomment when `/getting-started/download#package-managers` is available
  - [ ] Uncomment when `/getting-started/javascript#using-boosted-as-a-module` is available
  - [ ] Uncomment when `/content/reboot` is available (See #2851)
- [ ] **Getting Started > JavaScript**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/getting-started/javascript.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/getting-started/javascript.md))
  - [ ] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2648
- [ ] **Getting Started > Parcel**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/getting-started/parcel.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/getting-started/parcel.md))
- [ ] **Getting Started > RFS**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/getting-started/rfs.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/getting-started/rfs.md))
- [ ] **Getting Started > RTL**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/getting-started/rtl.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/getting-started/rtl.md))
- [ ] **Getting Started > Vite**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/getting-started/vite.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/getting-started/vite.md))
- [ ] **Getting Started > Webpack**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/getting-started/webpack.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/getting-started/webpack.md))
  - [ ] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2652 

### Orange Design System for web

- [ ] [**Orange Design System for web**](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/tree/main/site/content/docs/5.3/guidelines)
  - Add the link to the header 
  - Reintegrate or remove `site/data/design-guidelines.yml`
  - Possibly reintegrate `site/static/docs/**/assets/img/platforms/*` images, and mention them in the `NOTICE.txt`

### Helpers

- [ ] **Helpers > Clearfix**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/helpers/clearfix.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/helpers/clearfix.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2673
  - [ ] Uncomment example when button component is available
- [ ] **Helpers > Color background**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/helpers/color-background.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/helpers/color-background.md))
  - [ ] Handle comment whenever `/components/badge` is added
  - [ ] Handle comment whenever `/components/card` is added
- [ ] **Helpers > Colored links**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/helpers/colored-links.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/helpers/colored-links.md))
  - [ ] Handle comment whenever `/utilities/link` is added
- [ ] **Helpers > Focus ring**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/helpers/focus-ring.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/helpers/focus-ring.md))
- [ ] **Helpers > Icon link**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/helpers/icon-link.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/helpers/icon-link.md))
- [x] **Helpers > Position**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/helpers/position.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/helpers/position.md))
- [ ] **Helpers > Ratio**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/helpers/ratio.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/helpers/ratio.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2717
  - [ ] Uncomment elements whenever `/content/reboot` is added (See #2851)
- [ ] **Helpers > Stacks**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/helpers/stacks.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/helpers/stacks.md))
  - [ ] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2681 
  - [ ] Uncomment elements whenever `/helpers/vertical-rule` is added
  - [ ] Uncomment elements whenever buttons are developed
  - [ ] Uncomment elements whenever inputs are developed
- [ ] **Helpers > Stretched link**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/helpers/stretched-link.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/helpers/stretched-link.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2719
  - [ ] Uncomment elements whenever `/components/card` is added
  - [ ] Uncomment elements whenever `/components/buttons` is added
  - [x] Uncomment elements whenever `/layout/grid` is added
  - [x] Uncomment elements whenever `/utilities/spacings` is added
- [x] **Helpers > Text truncation**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/helpers/text-truncation.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/helpers/text-truncation.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2670
- [ ] **Helpers > Vertical rule**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/helpers/vertical-rule.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/helpers/vertical-rule.md))
  - [ ] Remove overwrite in homepage on masthead 
- [x] **Helpers > Visually hidden**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/helpers/visually-hidden.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/helpers/visually-hidden.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2674

### Layout

- [x] **Layout > Breakpoints**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/layout/breakpoints.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/layout/breakpoints.md))
- [x] **Layout > Columns**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/layout/columns.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/layout/columns.md))
- [x] **Layout > Containers**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/layout/containers.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/layout/containers.md))
- [x] **Layout > CSS Grid**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/layout/css-grid.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/layout/css-grid.md))
  - [x] #2679
  - [x] Uncomment responsive part when responsive documentation is ready
  - [x] Uncomment `--bs-gap` fallback value when value is set accordingly to token value
- [x] **Layout > Grid**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/layout/grid.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/layout/grid.md))
- [x] **Layout > Gutters**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/layout/gutters.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/layout/gutters.md))
- [ ] **Layout > Utilities**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/layout/utilities.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/layout/utilities.md))
  - [x] Uncomment "Margin and padding" when Dimension will be developed 
  - [ ] Handle comment whenever `/components/alerts` is added
- [ ] **Layout > Z-index**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/layout/z-index.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/layout/z-index.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2741
  - [ ] Uncomment when we'll have some more components (navigation, tooltips, popovers, modals, navbars, buttons, input group, etc.). The sentence mention a lot of them

### Utilities

- [ ] **Utilities > API**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/api.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/api.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2656
  - [x] "Responsive" section (+ responsive link in the table) when `.opacity-` utilities will be implemented
  - [ ] `rfs` link in the table when `/getting-started/rfs` page will be up
  - [ ] Link to update when `/utilities/text#text-decoration` will be up (See #2851)
  - [x] Link to update when `/utilities/opacity` will be up
  - [x] Link to update when `/utilities/position` will be up
  - [x] "CSS variable utilities" to uncomment when `.text-opacity` will be explained
  - [x] "Local CSS variables" to uncomment when `.bg-*` utilities will be explained
  - [x] "Add utilities" to uncomment
  - [x] "Modify utilities" to uncomment when width utilities will be explained
  - [x] "Enable responsive" to uncomment when border utilities will be explained and breakpoints reintroduced
  - [x] "Add utilities" to uncomment when pointer utilities will be explained and the file architecture is defined
  - [x] "Rename utilities" to uncomment when margin utilities will be explained and the file architecture is defined
  - [x] "Remove utilities" to uncomment when width utilities will be explained and the file architecture is defined
  - [x] "Add, remove, modify" to uncomment when width utilities will be explained and the file architecture is defined
- [ ] **Utilities > Background**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/background.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/background.md))
  - [ ] Handle comment whenever `/customize/css-variables` is added
- [ ] **Utilities > Borders**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/borders.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/borders.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2687
    - [ ] Remaining element to tackle: https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2687#discussion_r1760605151. Do we keep an `@if $enable-bootstrap-compatibility {` around the `border-radius` CSS variables? 
  - [x] Border colors
  - [x] Border opacity
  - [x] Dividers
- [x] **Utilities > Colors**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/colors.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/colors.md))
- [x] **Utilities > Display**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/display.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/display.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2659
  - [x] Uncomment things when responsive and breakpoints are defined
  - [x] Uncomment things when `.text-bg-*` utilities will be present
- [ ] **Utilities > Flex**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/flex.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/flex.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2660
  - [x] Uncomment elements when responsive will be explained in the documentation
  - [ ] **Asked to the designers, and waiting for an answer**: change the colored containers?
- [x] **Utilities > Float**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/float.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/float.md))
- [x] **Utilities > Interactions**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/interactions.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/interactions.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2662 
- [ ] **Utilities > Link**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/link.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/link.md))
- [ ] **Utilities > Object fit**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/object-fit.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/object-fit.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2663
  - [x] Uncomment elements whenever responsive mode (breakpoints and grids) is defined
  - [ ] If `rounded` class has actually a rounded rendering, add them like in Bootstrap 
- [ ] **Utilities > Opacity**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/opacity.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/opacity.md))
- [x] **Utilities > Overflow**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/overflow.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/overflow.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2664
- [ ] **Utilities > Position**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/position.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/position.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2666
  - [ ] Uncomment when button component will be developed
  - [ ] Uncomment when progress component will be developed
  - [x] Uncomment when border + bg colors utilities will be developed
- [ ] **Utilities > Shadows**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/shadows.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/shadows.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2741
  - [ ] Uncomment when we'll have components and/or `$enable-shadows`
- [x] **Utilities > Sizing**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/sizing.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/sizing.md))
- [x] **Utilities > Spacing**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/spacing.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/spacing.md))
  - [ ] Change the migration guide content related to Spacing to avoid listing all classes. Maybe thanks to a table listing the equivalent ? 
- [ ] **Utilities > Text**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/text.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/text.md))
- [x] **Utilities > Vertical align**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/vertical-align.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/vertical-align.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2668 
- [x] **Utilities > Visibility**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/visibility.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/visibility.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2651 
- [ ] **Utilities > Z-index**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/z-index.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/z-index.md))
  - [x] https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2669 
  - [ ] Uncomment things when the first components will be developed
  - [ ] Add `.rounded-3` like in Bootstrap if `.rounded` classes are authorized in OUDS Web

### Components

- [ ] **Components > Accordion**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/accordion.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/accordion.md))
- [ ] **Components > Alerts**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/alerts.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/alerts.md))
- [ ] **Components > Back to top**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/back-to-top.md))
- [x] **Components > Badge**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/badge.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/badge.md))
- [x] **Components > Breadcrumb**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/breadcrumb.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/breadcrumb.md))
- [ ] **Components > Button group**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/button-group.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/button-group.md))
- [x] **Components > Buttons**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/buttons.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/buttons.md))
- [ ] **Components > Card**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/card.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/card.md))
  - Reintegrate the corresponding examples 
- [ ] **Components > Carousel**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/carousel.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/carousel.md))
- [ ] **Components > Close button**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/close-button.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/close-button.md))
- [ ] **Components > Collapse**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/collapse.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/collapse.md))
- [ ] **Components > Dropdowns**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/dropdowns.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/dropdowns.md))
- [ ] **Components > Footer**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/footer.md))
  - Reintegrate the corresponding examples
- [ ] **Components > List group**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/list-group.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/list-group.md))
- [ ] **Components > Local navigation**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/local-navigation.md))
- [ ] **Components > Modal**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/modal.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/modal.md))
- [ ] **Components > Navbar**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/navbar.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/navbar.md))
- [ ] **Components > Nav & tabs**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/nav-tabs.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/nav-tabs.md))
- [ ] **Components > Offcanvas**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/offcanvas.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/offcanvas.md))
- [ ] **Components > Orange navbar**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/orange-navbar.md))
  - Do we reintegrate the OBS logos? If so, reintegrate `site/static/docs/**/assets/brand/OBS-*` and mention them in the `NOTICE.txt` file.
  - Possibly reintegrate `navbar-contact.png` and mention it in the `NOTICE.txt` file
  - Reintegrate the corresponding examples
- [ ] **Components > Pagination**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/pagination.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/pagination.md))
- [ ] **Components > Placeholders**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/placeholders.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/placeholders.md))
- [ ] **Components > Popovers**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/popovers.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/popovers.md))
- [ ] **Components > Progress**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/progress.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/progress.md))
- [ ] **Components > Scrollspy**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/scrollspy.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/scrollspy.md))
- [ ] **Components > Spinners**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/spinners.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/spinners.md))
- [ ] **Components > Stepped process**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/stepped-process.md))
- [ ] **Components > Sticker**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/sticker.md))
  - Reintegrate the corresponding examples
- [x] **Components > Tags**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/tags.md))
  - Reintegrate the corresponding examples
- [ ] **Components > Title bars**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/title-bars.md))
  - Reintegrate the corresponding examples
  - Possibility reintegrate `site/static/docs/**/assets/img/title-bars-illustrations/*` images and reintegrate them in the `NOTICE.txt` file 
- [ ] **Components > Toasts**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/toats.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/toats.md))
- [ ] **Components > Tooltips**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/components/tooltips.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/components/tooltips.md))

### Forms

- [x] **Forms > Check Radios**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/forms/checks-radios.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/forms/checks-radios.md))
- [ ] **Forms > Floating Labels**: ???? ([Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/forms/floating-labels.md))
- [ ] **Forms > Form control**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/forms/form-control.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/forms/form-control.md))
- [ ] **Forms > Input group**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/forms/input-group.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/forms/input-group.md))
- [ ] **Forms > Layout**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/forms/layout.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/forms/layout.md))
- [ ] **Forms > Overview**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/forms/overview.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/forms/overview.md))
- [ ] **Forms > Quantity selector**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/forms/quantity-selector.md))
- [ ] **Forms > Range**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/forms/range.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/forms/range.md))
- [x] **Forms > Select**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/forms/select.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/forms/select.md))
- [ ] **Forms > Validation**: ([Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/forms/validation.md) / [Bootstrap](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/forms/validation.md))

### Examples

> [!NOTE]  
> For all examples, some data are handled in [site/data/examples.yml](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/data/examples.yml)

> [!CAUTION]  
> As soon as the first example is reintegrated, please remove the "Fake" example from the source code

- [ ] [**Cards**](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/tree/main/site/content/docs/5.3/examples/cards) ([RTL](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/tree/main/site/content/docs/5.3/examples/cards-rtl))
- [ ] [**Download app**](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/tree/main/site/content/docs/5.3/examples/download-app)
  - Reintegrate all `site/content/docs/**/examples/download-app/img/` images and mention them in the `NOTICE.txt` file 
- [ ] [**Footer**](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/tree/main/site/content/docs/5.3/examples/footers)
- [ ] [**Form**](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/tree/main/site/content/docs/5.3/examples/form)
- [x] [**Grid**](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/tree/main/site/content/docs/5.3/examples/grid) ([Bootstrap](https://github.com/twbs/bootstrap/tree/main/site/content/docs/5.3/examples/grid))
- [ ] [**Masonry**](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/tree/main/site/content/docs/5.3/examples/masonry) ([Bootstrap](https://github.com/twbs/bootstrap/tree/main/site/content/docs/5.3/examples/masonry))
- [ ] [**Navbar mode selector**](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/tree/main/site/content/docs/5.3/examples/navbar-mode-selector)
- [ ] [**Navbar sticky**](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/tree/main/site/content/docs/5.3/examples/navbar-sticky)
- [ ] [**Navbars**](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/tree/main/site/content/docs/5.3/examples/navbars)
  - Do we reintegrate the OBS logos? If so, reintegrate `site/static/docs/**/assets/brand/OBS-*` and mention them in the `NOTICE.txt` file.
- [ ] [**Stickers**](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/tree/main/site/content/docs/5.3/examples/stickers)
- [ ] [**Tags**](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/tree/main/site/content/docs/5.3/examples/tags)
- [ ] [**Title bars**](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/tree/main/site/content/docs/5.3/examples/title-bars)

### First 0.1 version

- [ ] Prepare the versions select to handle minor and major versions
  - See [`site/data/docs-versions.yml`](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/data/docs-versions.yml)

### First version releases ready to be shared

- [ ] In `gh-pages`, double-check the robots.txt file to maybe remove this rule: `Disallow: /ouds-web/*`
- [ ] "Deprecated in" and "Added in" messages (see https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2588) to use
- [ ] Verify that all shortcodes are used, otherwise, delete them
- [ ] Verify that all partials are used, otherwise, delete them
  - WARN  Template _default/_markup/render-image.html is unused, source file /site/layouts/_default/_markup/render-image.html
  - WARN  Template partials/callouts/danger-async-methods.md is unused, source file /site/layouts/partials/callouts/danger-async-methods.md
  - WARN  Template partials/callouts/info-mediaqueries-breakpoints.md is unused, source file /site/layouts/partials/callouts/info-mediaqueries-breakpoints.md
  - WARN  Template partials/callouts/info-npm-starter.md is unused, source file /site/layouts/partials/callouts/info-npm-starter.md
  - WARN  Template partials/callouts/info-prefersreducedmotion.md is unused, source file /site/layouts/partials/callouts/info-prefersreducedmotion.md
  - WARN  Template partials/callouts/info-sanitizer.md is unused, source file /site/layouts/partials/callouts/info-sanitizer.md
  - WARN  Template partials/callouts/warning-color-assistive-technologies.md is unused, source file /site/layouts/partials/callouts/warning-color-assistive-technologies.md
  - WARN  Template partials/callouts/warning-data-bs-title-vs-title.md is unused, source file /site/layouts/partials/callouts/warning-data-bs-title-vs-title.md
  - WARN  Template partials/callouts/warning-input-support.md is unused, source file /site/layouts/partials/callouts/warning-input-support.md
  - WARN  Template partials/guide-footer.md is unused, source file /site/layouts/partials/guide-footer.md
  - WARN  Template partials/js-data-attributes.md is unused, source file /site/layouts/partials/js-data-attributes.md
  - WARN  Template partials/responsive-img.html is unused, source file /site/layouts/partials/responsive-img.html
  - WARN  Template partials/table-content.html is unused, source file /site/layouts/partials/table-content.html
  - WARN  Template shortcodes/added-in.html is unused, source file /site/layouts/shortcodes/added-in.html
  - WARN  Template shortcodes/bs-table.html is unused, source file /site/layouts/shortcodes/bs-table.html
  - WARN  Template shortcodes/callout-deprecated-dark-variants.html is unused, source file /site/layouts/shortcodes/callout-deprecated-dark-variants.html
  - WARN  Template shortcodes/deprecated-in.html is unused, source file /site/layouts/shortcodes/deprecated-in.html
  - WARN  Template shortcodes/design-callout-alert.html is unused, source file /site/layouts/shortcodes/design-callout-alert.html
  - WARN  Template shortcodes/docsref.html is unused, source file /site/layouts/shortcodes/docsref.html
  - WARN  Template shortcodes/enable-btn-close-tooltip.html is unused, source file /site/layouts/shortcodes/enable-btn-close-tooltip.html
  - WARN  Template shortcodes/js-dismiss.html is unused, source file /site/layouts/shortcodes/js-dismiss.html
  - WARN  Template shortcodes/markdown.html is unused, source file /site/layouts/shortcodes/markdown.html
  - WARN  Template shortcodes/partial.html is unused, source file /site/layouts/shortcodes/partial.html
  - WARN  Template shortcodes/placeholder.html is unused, source file /site/layouts/shortcodes/placeholder.html
  - WARN  Template shortcodes/svg-docs.html is unused, source file /site/layouts/shortcodes/svg-docs.html
  - WARN  Template shortcodes/table.html is unused, source file /site/layouts/shortcodes/table.html
- [ ] Double check the migration guide: 
  - [ ] All new breakpoints should be mentioned in the new utilities (e.g. in spacing, add the mention of new breakpoints maybe like `.m-{xs|sm|md|lg|xl|2xl|3xl}-smash` ?)
  - [ ] Do we add an Info tag to attract user's attention on `$enable-bootstrap-compatibility` possibilities like done with object fit utilities?
  - [ ] Decide whether we have a breakline between `<li>`s because the generated code is not totally the same (adding a `<p>` or not)

### After the renaming of the repository

- [ ] Composer and Packagist
  - The `name` property is already correct
  - `composer.json` will probably need to use the `replace` property
  - In the README, `composer require orange-opensource/orange-boosted-bootstrap:dev-ouds/main` could be changed to point to a real tagged version
  - In the README, the "packagist" badge should be reintegrated
- [ ] web.unified-design-system.orange.com must target `gh-pages` root level and boosted.orange.com must target `gh-pages/boosted` directory
